code/modules/admin/verbs/debug.dm

```diff
@@ -1244,7 +1244,18 @@ client/proc/check_convertables()
 		if(!chosen)
 			return
 
-	holder.marked_datum = new chosen
+	var/list/lst = list()
+	var/argnum = input("Number of arguments","Number:",0) as num|null
+	if(!argnum && (argnum!=0))
+		return
+
+	lst.len = argnum // Expand to right length
+
+	var/i
+	for(i = 1, i < argnum + 1, i++) // Lists indexed from 1 forwards in byond
```

Comment:
You can scrap the `lst.len = argnum` and `var/i` above and just do `for(var/i in 1 to argnum)` here and `lst += variable_set(src)` below

Reply:
True, I copypasted this from elsewhere without looking at it in too much details.